A32NX/html_ui/Pages/A32NX_Core/A32NX_Refuel.js

```diff
@@ -0,0 +1,154 @@
+const REFUEL_FACTOR = 0.42;
+const CENTER_MODIFIER = 3;
+
+class A32NX_Refuel {
+    constructor() {}
+
+    init() {
```

Comment:
Please add the `console.log` messages printed on initialization/init like other core modules